src/NetStreams/Internal/ConsumePipeline.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace NetStreams.Internal
 {
-    internal class ConsumePipeline<TKey, TMessage> : IConsumePipeline<TKey, TMessage>
+    public class ConsumePipeline<TKey, TMessage> : IConsumePipeline<TKey, TMessage>
```

Comment:
If we make this public then we need to remove it from the internal name space.

Reply:
Moved the class out of the Internal namespace.